validation-test/ParseableInterface/verify_stdlib.swift

```diff
@@ -1,6 +1,10 @@
 // Note that this test should still "pass" when Swift.swiftinterface has not
 // been generated.
 
+// SWIFT_ENABLE_TENSORFLOW
+// FIXME(TF-489): Re-enable this test after fixing `.swiftinterface` errors.
+// XFAIL: nonexecutable_test
```

Comment:
Will this cause a failure on Linux, if this test does not fail on Linux? I think that the test doesn't currently fail on Linux, because everything is currently passing for me on Linux.

Reply:
Ah, that's a great point.
Let me verify a change that passes on both Linux and macOS before retriggering CI.

Reply:
I tried this out. I do indeed get "unexpected success" on Linux. You could disable it with `UNSUPPORTED: nonexecutable_test` instead.

Reply:
Done, thanks!